packages/server/src/logger.ts

```diff
@@ -7,7 +7,7 @@ export interface LoggerOptions {
   out: <A extends unknown[]>(...args: A) => void;
 }
 
-export function* createLogger({ atom, out }: LoggerOptions) {
+export function* createBundlerLogger({ atom, out }: LoggerOptions) {
```

Comment:
this was always a really quick fix as there was nothing.  

The logging is a ticket all on its own

Reply:
As far as the state goes

The validation happens outside of the bundling in the manifest-generator which is like a preprocessor.  I did think about putting it in rollup but I think it would complicate things with little benefit.  The bundler works on the single entry point of `manifest.js` and you would need a complicated plugin to make it work for multiple entry points or test files.

So the state goes from `'GREEN'` back to `'VALIDATING'` when the `chokidar` watcher gets triggered.

Be great to have a finite state machine around this.  Might give an extra layer of structure to the code also.